src/index.html

```diff
@@ -7,12 +7,55 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__product">
+        <img
+          src="./images/imac.jpeg"
+          alt="img"
+          class="card__image"
+        />
+        <div class="card__conainer">
+          <p class="card__description">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+          <p class="card__product-code">Product code: 195434</p>
+
+          <div class="card__rating">
+            <div class="stars">
```

Comment:
```suggestion
            <div class="stars stars--4">
```
The stars quantity should depend on the modifier